styles/overrides/_modal.scss

```diff
@@ -32,8 +32,7 @@
   .close {
     position: absolute;
     right: $layout-spacing-base;
-    top: 50%;
-    transform: translateY(-50%);
+    top: $layout-spacing-base;
```

Comment:
💚 It's worth checking that this doesn't bork any of the other current modal instances in the App